src/README.md

```diff
@@ -0,0 +1,136 @@
+# API setup:
+
+## Mongo Database setup:
+
+- Create MongoDB account.
+
+- Create Database cluster.
+
+- Create User collection.
+
+- In the "Database Deployments" overview, press on "Connect".
+
+![alt](https://i.imgur.com/A0Dwk6F.png)
+
+- In the popup press on "Connect your application" and then you will be greeted with the screen which will have connection string defined for you:
+
+![alt](https://i.imgur.com/uUSSwMD.png)
+
+In the above example, "user" is username who has an access to your database cluster, and password is theirs password.
+
+- To register new user who will have an access to the cluster (admin user because), go to the "Database Access" and press on "Add new database user".
+
+![alt](https://i.imgur.com/x6CyUzw.png)
+
+Save password of that user and their username, you will use it in your connection string.
+
+- Finally, in "Network Access" change IP address to 0.0.0.0 to make your Database accessible by any IP (anyone who has connection string with correct user credentials you defined in previous step).
+
+![alt](https://i.imgur.com/A4a8mbQ.png)
+
+- In the project folder, create ".env" file and fill it with your information. Example is here:
+
+![alt](https://i.imgur.com/msHT5eu.png)
+
+environment variable names should be the same, the only parts that change are values.
+
+## Firebase server setup:
```

Comment:
Same question here, do we all have to do this?

Reply:
- Same answer as to the previous question